app/src/interfaces/translations/translations.vue

```diff
@@ -241,7 +241,9 @@ function useLanguages() {
 			}
 
 			if (!secondLang.value) {
-				secondLang.value = languages.value[1]?.[pkField];
+				const defaultLocale = userLanguage.value ? defaultLanguage.value : null;
+				const lang = languages.value.find((lang) => lang[pkField] === defaultLocale) || languages.value[0];
```

Comment:
Doesn't this find the first language where the language is the same as the userLanguage, just like `firstLang`? 🤔

Reply:
Ohh no nevermind! It's defaulting to the defaultLanguage at the opposite codepath 🙂